api/src/pcapi/core/offers/api.py

```diff
@@ -930,7 +946,10 @@ def reject_inappropriate_products(ean: str) -> bool:
         extra={"ean": ean, "products": [p.id for p in products], "offers": offer_ids},
     )
 
-    search.async_index_offer_ids(offer_ids)
+    if len(offer_ids) > 0:
```

Comment:
(détail / style) Plus courant en Python : 

```suggestion
    if offers:
```